scripts/externalize_js.sh

```diff
@@ -78,14 +78,17 @@ git commit -m "first pass externalizing javascript in $module"
 
 
 # a bit more yelling at the user
+# (maybe should just open these in an available browser? todo)
```

Comment:
This sounds hard, as these are usually URL patterns rather than actual URLs - you'd need a domain to use, and maybe some sort of UUID